temporian/core/test/utils.py

```diff
@@ -200,3 +201,80 @@ def __init__(self, event: Event):
 
 
 operator_lib.register_operator(OpI1O2)
+
+
+class OpWithAttributes(base.Operator):
+    @classmethod
+    def build_op_definition(cls) -> pb.OperatorDef:
+        return pb.OperatorDef(
+            key="OpWithAttributes",
+            inputs=[
+                pb.OperatorDef.Input(key="event"),
+            ],
+            outputs=[
+                pb.OperatorDef.Output(key="output"),
+            ],
+            attributes=[
+                pb.OperatorDef.Attribute(
+                    key="attr_int",
+                    type=pb.OperatorDef.Attribute.Type.INTEGER_64,
+                    is_optional=False,

```

Comment:
Nit: is_optional is optional. The default value is False.

Reply:
That's a self explanatory var name 😆